internal/services/loganalytics/log_analytics_linked_service_resource.go

```diff
@@ -365,6 +350,19 @@ func resourceLogAnalyticsLinkedServiceSchema() map[string]*pluginsdk.Schema {
 	}
 
 	if !features.ThreePointOhBeta() {
+		out["workspace_id"] = &pluginsdk.Schema{
+			Type:             pluginsdk.TypeString,
+			Computed:         true,
+			Optional:         true,
+			DiffSuppressFunc: suppress.CaseDifference,
+			ValidateFunc:     azure.ValidateResourceID,
+			ExactlyOneOf: func() []string {
+				if !features.ThreePointOhBeta() {
+					return []string{"workspace_id", "workspace_name"}
+				}
+				return []string{}
+			}(),
```

Comment:
Since we're in the if block anyway we could get rid of the function ? 
```suggestion
			ExactlyOneOf:  []string{"workspace_id", "workspace_name"},
```